### PR TITLE
update repository hostname

### PR DESCRIPTION
**Pull request by ureyni**
The repo hostname has been updated to the product name.

### CODE REVIEW

File: vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SunnyValley: {
   fingerprints: "/usr/local/etc/pkg/fingerprints/SunnyValley",
-  url: "https://updates.sunnyvalley.io/opnsense/${ABI}/%%PLUGIN_ABI%%/OpenSSL/latest",
+  url: "https://updates.zenarmor.com/opnsense/${ABI}/%%PLUGIN_ABI%%/OpenSSL/latest",
```
Comment (fichtner) [inline vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in:3]: Theoretically you could also drop “OpenSSL” now while at it?

Comment (ureyni) [reply vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in]: yes your right, I removed OpenSSL path

Comment (fichtner) [reply vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in]: Thanks. Is this going to be ready for release in 23.7.5? Not sure if next week or the week after.

Comment (ureyni) [reply vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in]: We are ready, we are performing final checks.

Comment (fichtner) [reply vendor/sunnyvalley/src/etc/pkg/repos/SunnyValley.conf.in]: Ok, if something goes wrong let me know else I’ll pick it up as discussed. Thanks for your work 😊

**Pull request merged**